kanidm_book/src/pam_and_nsswitch.md

```diff
@@ -39,7 +39,9 @@ You can check the privileged tasks daemon is running with
 > it provides supporting kanidm's capabilities.
 
 Both unixd daemons use the connection configuration from /etc/kanidm/config. This is the covered in
-client_tools. You can also configure some details of the unixd daemons in /etc/kanidm/unixd.
+[client_tools](./client_tools.md#kandim-configuration). 
+
+You can also configure some details of the unixd daemons in /etc/kanidm/unixd.
```

Comment:
I think this text should be "you can also configure some unixd specific options with the file ..."